### ISSUE TITLE
Build fails under iojs 2.0.0

### ISSUE THREAD

**Issue by ccheever** [justmoon/node-bignum#47]
Something changed with the latest version of iojs that makes the bignum module not build.
It looks like something just changed with a function signature in v8. Probably a small fix.

```
ccheever@Charlies-MacBook:~/tmp$npm install bignum
-
> bignum@0.9.2 install /Users/ccheever/tmp/node_modules/bignum
> node-gyp configure build

  CXX(target) Release/obj.target/bignum/bignum.o
In file included from ../bignum.cc:8:
In file included from ../node_modules/nan/nan.h:74:
In file included from ../node_modules/nan/nan_new.h:190:
../node_modules/nan/nan_implementation_12_inl.h:181:66: error: too many
      arguments to function call, expected at most 2, have 4
  return v8::Signature::New(v8::Isolate::GetCurrent(), receiver, argc, argv);
         ~~~~~~~~~~~~~~~~~~                                      ^~~~~~~~~~
/Users/ccheever/.node-gyp/2.0.0/deps/v8/include/v8.h:4188:3: note: 'New'
      declared here
  static Local<Signature> New(
  ^
1 error generated.
make: *** [Release/obj.target/bignum/bignum.o] Error 1
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/Users/ccheever/.nvm/versions/io.js/v2.0.0/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:269:23)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:169:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:1009:12)
gyp ERR! System Darwin 14.3.0
gyp ERR! command "/Users/ccheever/.nvm/versions/io.js/v2.0.0/bin/iojs" "/Users/ccheever/.nvm/versions/io.js/v2.0.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "configure" "build"
gyp ERR! cwd /Users/ccheever/tmp/node_modules/bignum
gyp ERR! node -v v2.0.0
gyp ERR! node-gyp -v v1.0.3
gyp ERR! not ok
npm ERR! Darwin 14.3.0
npm ERR! argv "/Users/ccheever/.nvm/versions/io.js/v2.0.0/bin/iojs" "/Users/ccheever/.nvm/versions/io.js/v2.0.0/bin/npm" "install" "bignum"
npm ERR! node v2.0.0
npm ERR! npm  v2.9.0
npm ERR! code ELIFECYCLE

npm ERR! bignum@0.9.2 install: `node-gyp configure build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the bignum@0.9.2 install script 'node-gyp configure build'.
npm ERR! This is most likely a problem with the bignum package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp configure build
npm ERR! You can get their info via:
npm ERR!     npm owner ls bignum
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/ccheever/tmp/npm-debug.log
ccheever@Charlies-MacBook:~/tmp$
```

**Comment by indutny** [comment]
Confirmed:

```
../node_modules/nan/nan_implementation_12_inl.h:181:66: error: too many arguments to function call, expected at most 2, have 4
  return v8::Signature::New(v8::Isolate::GetCurrent(), receiver, argc, argv);
         ~~~~~~~~~~~~~~~~~~                                      ^~~~~~~~~~
```

**Comment by rvagg** [comment]
fixed @ v0.9.3

**Comment by indutny** [comment]
Thank you!
